src/tasks-handler.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-plusplus */
```

Comment:
You should refrain to disable linters. To correctly avoid this error you can do `var += 1` instead of `var++`